sentinelsat/sentinel.py

```diff
@@ -209,7 +209,9 @@ def format_query(area=None, date=None, raw=None, area_relation="Intersects", **k
             date_attrs = ["beginposition", "endposition", "date", "creationdate", "ingestiondate"]
             if attr.lower() in date_attrs:
                 # Automatically format date-type attributes
-                if isinstance(value, string_types) and " TO " in value:
+                if isinstance(value, set):
```

Comment:
@Fernerkundung and @valgur - only `set` should be coerced to `OR`? Not `tuple` and `list`, too?

Reply:
I guess it makes most sense for `set`, because it has unique elements and is unordered.

Reply:
In the current state only sets are coerced as `OR`. Everything else gets parsed as `START TO END`.

Do we also want to support: `START1 TO END1 OR START2 TO END2`? That would require some changes to the logic. However, I can't see a compelling use case for this that can't be solved with 2 queries.

Reply:
no, fine this way then.